File: src/test/java/io/plagov/adven/aoc2016/Day02Test.java
package io.plagov.adven.aoc2016;

import io.plagov.advent.aoc2016.Day02;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;
import util.FileUtilKt;

import java.util.List;

class Day02Test {

    private final List<String> input = FileUtilKt.readInputFile("2016/day02.txt");

    @Test
    void solveFirst() {
        var day2 = new Day02();
        Assertions.assertEquals("14894", day2.partOne(input));
    }
}

File: src/test/kotlin/io/plagov/advent/aoc2015/Day02Test.kt
package io.plagov.advent.aoc2015

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test
import util.readInputFile

class Day02Test {

  private val day2 = Day02()

  private val taskInput = readInputFile("2015/day02.txt")

  @Test
  fun solveFirstForTaskInput() {
    assertEquals(1_588_178, day2.partOne(taskInput))
  }
}
